insalan/tournament/admin.py

```diff
@@ -461,3 +466,214 @@ def get_tournament(self, obj):
 
 
 admin.site.register(Substitute, SubstituteAdmin)
+
```

Comment:
Filtering is already implemented via the list_filter attribute. But I agreed it could be improved for round index and index in round.

Reply:
Yes, it's what list_filter attribute is for '^^. I just did not provide custom filters because it is note needed in most cases but I will do it for round index and index in round because number are not that clear.